libmachine/provision/debian.go

```diff
@@ -44,7 +44,7 @@ func (provisioner *DebianProvisioner) Service(name string, action pkgaction.Serv
 		return err
 	}
 
-	command := fmt.Sprintf("sudo systemctl %s %s", action.String(), name)
+	command := fmt.Sprintf("sudo systemctl %s %s || true", action.String(), name)
```

Comment:
I wasn't able to reproduce, but @ehazlett claimed `systemctl enable docker` was failing for him with `Failed to execute operation: File exists` because we're already enabled.  When I run that, I get a lot of harmful looking output, but it exits appropriately with a zero exit code. :confused: